ui/src/app/job-list/job-list.component.ts

```diff
@@ -32,6 +32,7 @@ export class JobListComponent implements OnInit {
   // whenever we change query parameters, via a subscription.
   jobs = new BehaviorSubject<JobListView>({
     results: [],
+    totalKnownResults: 0,
```

Comment:
nit: If it's valid for this to be null/undefined, then I'd probably start with that (just change the type in `JobListView` to `number|null` or `number|undefined`).